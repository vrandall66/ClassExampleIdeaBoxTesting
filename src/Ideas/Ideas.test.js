import React from "react";
import { shallow } from "enzyme";
import Ideas from "./Ideas";

describe("Ideas", () => {
  it("Should match the snapshot with the data passed in correctly", () => {
    let ideas = [
      { title: "Bananas", description: "BANANAS", id: 4 },
      { title: "Orange", description: "ORANGE", id: 5 }
    ];
    const wrapper = shallow(<Ideas ideas={ideas} removeIdea={jest.fn()} />);
    expect(wrapper).toMatchSnapshot();
  });
});
