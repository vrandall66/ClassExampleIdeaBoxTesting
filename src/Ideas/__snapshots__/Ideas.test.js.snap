// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ideas Should match the snapshot with the data passed in correctly 1`] = `
<Fragment>
  <h2>
    Ideas Component
  </h2>
  <section
    className="ideas"
  >
    <Card
      description="BANANAS"
      id={4}
      key="4"
      removeIdea={[MockFunction]}
      title="Bananas"
    />
    <Card
      description="ORANGE"
      id={5}
      key="5"
      removeIdea={[MockFunction]}
      title="Orange"
    />
  </section>
</Fragment>
`;
